builders/build/templates.md

```diff
@@ -0,0 +1,130 @@
+- **[XCM](https://wiki.polkadot.network/docs/learn-xcm){target=_blank}** - handles the ingestion and dispatch of incoming downward and lateral messages, allowing a ContainerChain to communicate and interoperate with the other sibling chains within the ecosystem
+- **Runtime Upgrades** - a runtime upgrade in a ContainerChain must be informed to the relay chain to allow its validators to check on the blocks produced by the collators of the ContainerChains. Cumulus notifies the upgrade to the relay chain and waits the required amount of time (blocks) before enacting the change
+
+More information about how to configure Cumulus SDK to integrate an Appchain into the Polkadot's ecosystem can be found in the [official Cumulus template](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/parachain-template).
+
+## Base Setup - Supporting the Tanssi Protocol {: #base-setup-supporting-tanssi }
+
+Besides Cumulus, a ContanerChain must implement the Tanssi modules to support the protocol and benefit safely from Tanssi's block production as a service:
+
+- **Authorities Noting** - registers the set of collators assigned to the ContainerChain by Tanssi
+- **Author Inherent** - Allows the collator authoring the block to include its identity to get validated and rewarded
+
+If you don't include these modules in the ContainerChain's runtime, there won't be a method to confirm that the blocks are being generated by trustworthy collators designated by the Tanssi orchestrator. This could create a vulnerability for malicious actors to exploit and compromise the ContainerChain.
+
+## Baseline Appchain template {: #baseline-appchain-template }
+
+Developing a Substrate-based runtime typically involves two main steps: first, [incorporating pre-existing Substrate built-in modules](XXX){target=_blank} into the runtime, and second, [crafting new modules](XXX){target=_blank} tailored to the specific application.
+
+Since the template already contains the necessary configuration for seamless integration into the Polkadot ecosystem and for Tanssi protocol compatibility, teams interested in constructing an innovative Appchain can use this template as a starting point to start adding their custom logic.
+
+## Baseline EVM (Ethereum Virtual Machine) Template {: #baseline-evm-template }
+
+For teams developing their applications on top of an EVM (Ethereum Virtual machine), this template is a foundational starting point. It contains all the essential references to add the extra layer of Ethereum compatibility to a Substrate node: 
+
+- **EVM** - this referece adds the execution layer for Ethereum apps
+- **Ethereum** - this reference adds the Ethereum block production emulation to allow the RPC nodes (and DApps) to run without any modification
+- **EVMChainId** - stores the chain identifier that identifies the Ethereum network
```

Comment:
```suggestion
- **EVM Chain ID** - stores the chain identifier that identifies the Ethereum network
```

Reply:
Unless this is the name of the module. But if it is I would add that before the "Substrate node"

INstea dof "essential references I would put essencial modules"

Reply:
Yes, those are pallets. Added modules instead of references and also backticks